charts/prefect-orion/values.yaml

```
@@ -130,6 +130,11 @@ postgresql:
     # -- Version tag, corresponds to tags at https://hub.docker.com/r/bitnami/postgresql/
     tag: 14.3.0
 
+# Values placed here will result in environment variable definitions
+# starting with "PREFECT_ORION_" and will be passed to relevant
+# services
+prefectConfig:
```

```suggestion
prefectConfig: {}
```

if you use an empty object as a default, then it simplifies some of the logic - similar to what we're doing for annotations

the type info shown by helm-docs also seems weird (string instead of object?)

```
| prefectConfig | string | `nil` |  |
| serviceAccount.annotations | object | `{}` |  |
```